src/components/avatars/OcAvatarItem.vue

```diff
@@ -0,0 +1,133 @@
+<template>
+  <span
+    class="oc-avatar-item"
+    :style="{ backgroundColor: backgroundColor, '--icon-color': iconColor }"
+    :aria-label="accessibleLabel === '' ? null : accessibleLabel"
+    :aria-hidden="accessibleLabel === '' ? 'true' : null"
+    :focusable="accessibleLabel === '' ? 'false' : null"
+    :role="accessibleLabel === '' ? null : 'img'"
+    :data-test-link-name="name"
+  >
+    <oc-icon v-if="hasIcon" :name="icon" size="small"></oc-icon>
+  </span>
+</template>
+<script>
+export default {
+  name: "OcAvatarItem",
+  status: "ready",
+  release: "2.1.0",
```

Comment:
Please set `status` to `review` and release to an empty value. Will be updated during release.